src/main/java/io/vertx/serviceproxy/generator/ServiceProxyHandlerGen.java

```diff
@@ -0,0 +1,267 @@
+package io.vertx.serviceproxy.generator;
+
+import io.vertx.codegen.DataObjectModel;
+import io.vertx.codegen.Generator;
+import io.vertx.codegen.ParamInfo;
+import io.vertx.codegen.PropertyInfo;
+import io.vertx.codegen.type.*;
+import io.vertx.serviceproxy.generator.model.ProxyMethodInfo;
+import io.vertx.serviceproxy.generator.model.ProxyModel;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.AbstractMap.SimpleEntry;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * @author <a href="http://slinkydeveloper.github.io">Francesco Guardiani @slinkydeveloper</a>
+ */
+public class ServiceProxyHandlerGen extends Generator<ProxyModel> {
+
+  final GeneratorUtils utils;
+  final Map<String, String> numericMapping = Stream.of(
```

Comment:
make this static and just use a static block with a plain map put instead of convoluted creation for a simple map